Formula/f/flang.rb

```diff
@@ -0,0 +1,55 @@
+class Flang < Formula
+  desc "LLVM Fortran Frontend"
+  homepage "https://flang.llvm.org/"
+  url "https://github.com/llvm/llvm-project/releases/download/llvmorg-19.1.0/llvm-project-19.1.0.src.tar.xz"
+  sha256 "5042522b49945bc560ff9206f25fb87980a9b89b914193ca00d961511ff0673c"
+  # The LLVM Project is under the Apache License v2.0 with LLVM Exceptions
+  license "Apache-2.0" => { with: "LLVM-exception" }
+  head "https://github.com/llvm/llvm-project.git", branch: "main"
+
+  livecheck do
+    url :stable
+    regex(/^llvmorg[._-]v?(\d+(?:\.\d+)+)$/i)
+  end
+
+  depends_on "cmake" => :build
+  depends_on "llvm"
+  depends_on "zstd"
+  uses_from_macos "zlib"
+
+  def llvm
+    Formula["llvm"]
+  end
+
+  def install
+    # NOTE: Setting `BUILD_SHARED_LIBRARIES=ON` causes the just-built flang to throw ICE.
+    args = %W[
+      -DCLANG_DIR=#{llvm.opt_lib}/cmake/clang
+      -DCMAKE_INSTALL_RPATH=#{rpath}
```

Comment:
I don't think this is needed if `flang` libraries are static
```suggestion
```

Reply:
Yes, good point. I had that in there when I had `BUILD_SHARED_LIBS=ON`.